apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/issues/IssuesPage/IssuesPage.jsx

```diff
@@ -351,25 +457,14 @@ function IssuesPage(){
                 />
 
             
-            : <GithubServerTable
-                    key={key}
-                    headers={headers}
-                    resourceName={resourceName} 
-                    appliedFilters={appliedFilters}
-                    sortOptions={sortOptions}
-                    disambiguateLabel={disambiguateLabel}
-                    selectable = {true}
-                    loading={loading}
-                    fetchData={fetchData}
-                    filters={filtersOptions}
-                    promotedBulkActions={promotedBulkActions}
-                    hideQueryField={true}
-                    getNextUrl={getNextUrl}
-                    getStatus={func.getTestResultStatus}
-                    filterStateUrl={"/dashboard/issues"}
-                />
+            : components
             ]}
-            primaryAction={<Button primary onClick={() => openVulnerabilityReport()} disabled={showEmptyScreen}>Export vulnerability report</Button>}
+            primaryAction={
+                <HorizontalStack gap={2}>
+                    <DateRangeFilter initialDispatch={currDateRange} dispatch={(dateObj) => dispatchCurrDateRange({ type: "update", period: dateObj.period, title: dateObj.title, alias: dateObj.alias })} />
```

Comment:
secondary action{date range filter}, dont add in primary, primary is export only